src/App.js

```diff
@@ -1,23 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import Topics from "./components/Topics";
+import Header from "./components/header";
+import Articles from "./components/Articles";
+import Article from "./components/Article";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Routes>
+        <Route path="/" element={<Topics />} />
```

Comment:
You have "<Topics>" has your home path of '/' but there is no link to view all articles in this component so I can't see all of your articles like it mentions in the ticket